### PR TITLE
Replaced gql syntax with tql syntax to provide highlighting for all TypeQL keywords

### PR DESCRIPTION
**Pull request by opa-bear**
Also rebranded everything to TypeDB/TypeQL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Graql
-Syntax highliter for graql
+# TypeQL
+Syntax highliter for TypeQL
```
Comment (alexjpwalker) [inline README.md]:
```suggestion
Syntax highlighter for TypeQL
```

File: grammar/TypeQL.g4 [hunk 2]
```diff
@@ -0,0 +1,471 @@
+/*
```
Comment (alexjpwalker) [inline grammar/TypeQL.g4:1]: Let's revert the formatting of this file to make its content **precisely** identical to https://github.com/vaticle/typeql/blob/master/grammar/TypeQL.g4. This makes it easier to verify that the grammar is up-to-date.

image

File: package.json [hunk 3]
```diff
@@ -15,35 +15,35 @@
...
     ],
     "contributes": {
         "languages": [
             {
-                "id": "gql",
+                "id": "tql",
                 "aliases": [
-                    "Graql",
-                    "gql"
+                    "TypeDB",
```
Comment (alexjpwalker) [inline package.json]:
```suggestion
                    "TypeQL",
```

File: syntaxes/.vsixmanifest [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Language="en-US" Id="tql" Version="0.3.0" Publisher="fuzzycoder"/>
+    <DisplayName>TypeQL Syntax</DisplayName>
+    <Description xml:space="preserve">TypeQL Syntax Highlighter</Description>
+    <Tags>tql, typedb, typeql, gql,graql,grakn,Graql,__ext_gql</Tags>
```
Comment (alexjpwalker) [inline syntaxes/.vsixmanifest]: We have permanently abandoned the "Grakn" and "Graql" monikers so they shouldn't be referenced anywhere
```suggestion
    <Tags>tql, typedb, typeql, __ext_tql</Tags>
```

**Review by alexjpwalker — CHANGES_REQUESTED** [review]
Looks good against a few query types I've tried! I've requested some small changes.

File: grammar/TypeQL.g4 (hunk 2 above)
Comment (opa-bear) [reply grammar/TypeQL.g4:1]: Alex - perhaps this file should not even be in this repo as the single source of truth should be https://github.com/vaticle/typeql/blob/master/grammar/TypeQL.g4.  Thoughts?

Comment (alexjpwalker) [reply grammar/TypeQL.g4:1]: Yeah I agree, if this repo doesn't use this file then it shouldn't be here 👍

**Review by alexjpwalker — APPROVED** [review]

**Pull request merged**